batch/batch/batch.py

```diff
@@ -868,7 +868,7 @@ def from_record(record, deleted=False):
         return batches
```

Comment:
There was a job create row trigger that calculated essentially calculated the number of jobs.  Jackie set the number of jobs explicitly, and made the trigger unnecessary.  A nice simplification that could applied to batch1 (no matter how short-lived it will be).

Reply:
It had to be changed because we use the same aioclient for batch1 and batch2. The validator complained about the extra field, so I decided to just make the change because it was a good one in both places.